lib/her/model/orm.rb

```
@@ -75,6 +75,7 @@ def destroy
         method = self.class.method_for(:destroy)
         run_callbacks :destroy do
           self.class.request(:_method => method, :_path => request_path) do |parsed_data, response|
+            raise Her::Errors::RemoteResourceError if response.status >= 400
```

Yep! So since I caught up @samandmoore on our new strategy for only returning 204's from #destroy in TradingApi, we can get away with swallowing the validation errors.